packages/docusaurus-plugin-content-docs/src/versions/files.ts

```diff
@@ -28,27 +28,31 @@ function addPluginIdPrefix(fileOrDir: string, pluginId: string): string {
     : `${pluginId}_${fileOrDir}`;
 }
 
-/** `[siteDir]/community_versioned_docs/version-1.0.0` */
+/** `[siteDir]/[versionedDocsPath]/[community_versioned_docs/version-1.0.0` */
 export function getVersionDocsDirPath(
   siteDir: string,
   pluginId: string,
+  versionedDocsPath: string | undefined,
   versionName: string,
 ): string {
   return path.join(
     siteDir,
+    versionedDocsPath || '',
     addPluginIdPrefix(VERSIONED_DOCS_DIR, pluginId),
     `version-${versionName}`,
   );
 }
 
-/** `[siteDir]/community_versioned_sidebars/version-1.0.0-sidebars.json` */
+/** `[siteDir]/[versionedDocsPath]/community_versioned_sidebars/version-1.0.0-sidebars.json` */
```

Comment:
This comment assumes the path will be relative. Maybe it will be absolute in which case we don't need the `[siteDir]` prefix. 

Maybe the `siteDir` does not need anymore to be passed to this function? `path.join(versionedDocsPath,...rest)` could be enough?